app/models/manageiq/providers/autosde/storage_manager/physical_storage.rb

```diff
@@ -10,6 +13,17 @@ def raw_delete_physical_storage
     EmsRefresh.queue_refresh(ems)
   end
 
+  def raw_update_physical_storage(_options = {})
```

Comment:
Leading underscore indicates to linters to ignore a variable that isn't used later on, `_options` is used so better to have it just be `options = {}`

Reply:
fixed